crates/driver/src/infra/config/file/mod.rs

```diff
@@ -301,7 +301,7 @@ struct LiquidityConfig {
     #[serde(default)]
     zeroex: Option<ZeroExConfig>,
 
-    /// The base URL used to connect to subgraph clients.
```

Comment:
This argument would need to be moved into the `balancerv2` and `uniswapv3` config loading logic.